### PR TITLE
fix(docs): Fix typo in admin permission name

### PR DESCRIPTION
**Pull request by benedekh**
## To be verified by reviewers

* [ ] Relevant public API members have been documented
* [ ] Documentation related to this PR is complete
  * [ ] Boundary conditions are documented
  * [ ] Exceptions are documented
  * [ ] Nullability is documented if used
* [ ] Touched existing code has been extended with documentation if missing
* [ ] Code is readable
* [ ] New features and fixed bugs are covered by tests

### CODE REVIEW

File: docs/global/modules/core/pages/reference/component-model-server.adoc [hunk 1]
```diff
@@ -100,6 +100,6 @@ To enable it you can specify the following environment variables.
 
 The `permissions` claim of the token is expected to list directly granted permission.
 Additional granted permissions may be derived from that list.
-For example, an `admin` permission on a repository will grant write permission on all branches.
+For example, an `model-server/admin` permission on a repository will grant write permission on all branches.
```
Comment (odzhychko) [inline docs/global/modules/core/pages/reference/component-model-server.adoc:103]: This is talking about admin permissions for a repository. e.g. `repository/someRepositoryId/admin`

Comment (odzhychko) [inline docs/global/modules/core/pages/reference/component-model-server.adoc:103]:
```suggestion
For example, an `model-server/admin` permission gives access to all repositories and administrative action. `repository/someRepository/admin` on the other hand gives only permission to do everything in the repository "someRepository".
```

**Review by odzhychko — APPROVED** [review]

File: docs/global/modules/core/pages/reference/component-model-server.adoc (hunk 1 above)
Comment (benedekh) [reply docs/global/modules/core/pages/reference/component-model-server.adoc]: [f4b4161](https://github.com/modelix/modelix.core/pull/959/commits/f4b4161e9ad1948c657684af2602cb975bd29746) fixed this. Thank you for the fix!

**Pull request merged**